### PR TITLE
tweak tqdm to have download speed in kB/MB/etc

### PR DESCRIPTION
**Pull request by djsutherland**
This changes the progress bars in `_download_url_to_file` from saying things like `49773343.40it/s` to `47.5MB/s`.

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
Waiting on CI before land

**Comment by dougalsutherland** [comment]
It crashed in `test_structseq_repr`; don't see how that could possibly be related....

**Comment by soumith** [comment]
that's unrelated

**Pull request closed without merging**
